src/translations/en.json

```
@@ -0,0 +1,17 @@
+{
+	"Sort Ascending": "Sort Ascending",
+	"Sort Descending": "Sort Descending",
+	"Reset sorting": "Reset sorting",
+	"Remove column": "Remove column",
+	"No Data": "No Data",
+	"{count} cells copied": {
+		"0": "{count} cells copied",
+		"1": "{count} cell copied",
+		"default": "{count} cells copied"
+	},
+	"{count} rows selected": {
+		"0": "{count} rows selected",
+		"1": "{count} row selected",
+		"default": "{count} rows selected"
+	}
```

```suggestion
	"{count} cells copied": {
		"1": "{count} cell copied",
		"default": "{count} cells copied"
	},
	"{count} rows selected": {
		"1": "{count} row selected",
		"default": "{count} rows selected"
	}
```

Translation for 0 is the same as default.